freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/flow/SaltConfigProvider.java

```diff
@@ -57,6 +58,16 @@ public SaltConfig getSaltConfig(Stack stack, Set<Node> hosts) {
         return saltConfig;
     }
 
+    private static Map<String, Object> getPlatformMetadata(Stack stack) {
+        boolean govCloud = Boolean.FALSE;
```

Comment:
Why not the primitive `false`?

Reply:
I like it because easier to catch with my eyes due to the uppercase part :) but only my personal preference no convention reason behind this :smile:

Reply:
actually the default value is false, so you don't have to initialize